src/react/components/pages/projectSettings/projectSettingsPage.tsx

```diff
@@ -67,14 +67,11 @@ export default class ProjectSettingsPage extends React.Component<IProjectSetting
         const projectToUpdate: IProject = {
             ...formData,
         };
-
-        const isNew = !(!!projectToUpdate.id);
```

Comment:
I would still like to keep this post-save logic of redirection.

Reply:
@elizabethhalper  - Still seeing this code as removed... i don't think this should be changing with this bug fix.